### PR TITLE
New vends

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
- Moved few files that needed to be under _NF folders to correct locations, few textures, few yml and few ftl.
- Added a new machine called the CuddlyCritterVend with fun stuff in it.
- Added the "Jester Plushie", could be bought after machine emag.
- Added new items flag called "MarketPrice" to help set prices to some items that should keep a shop price as labeled without looking into the static or machine multi.
- Used the flag on all food items to make them lower while they still sell for less, added a lot more items for food machines (Most 3 items are now 30).
- Added 2 Eyepatchs for cargo pirate crate

### CODE REVIEW

**Comment by Cheackraze** [comment]
i kind of disagree with the food pricing change and the massive increase to food vendors. it turns the chef/cook gameplay into meaningless and worthless play.  With food being worth its nutrition value, at least chefs can make money through the game market if they cook a bunch, and I may even buff the value of frontier's custom Flavorol ingredient still as well.  Right now cakes sell for 250 a piece around flat price. of course to the depot more etc., but that goes a long way in 'baking' being worth it if you can still bake 2-3 dozen cakes and even if players don't buy, you can turn a profit etc.

**Comment by Cheackraze** [comment]
the rest of it is great work though, if you split it we can discuss that change in a different PR and I would merge the other vending machine stuff and we can discuss the hard set price changes separate too.

**Comment by dvir001** [comment]
> i kind of disagree with the food pricing change and the massive increase to food vendors. it turns the chef/cook gameplay into meaningless and worthless play. With food being worth its nutrition value, at least chefs can make money through the game market if they cook a bunch, and I may even buff the value of frontier's custom Flavorol ingredient still as well. Right now cakes sell for 250 a piece around flat price. of course to the depot more etc., but that goes a long way in 'baking' being worth it if you can still bake 2-3 dozen cakes and even if players don't buy, you can turn a profit etc.

I can up the food costs closer to what they were, or back to what they were. 
right now, the prices on that PR are either more logical or a bit more than the high sell price if you try to sell them.
The amounts of items can be lower again to around 5~ instead of 3 if it's fine with you, I was trying to go with the "Vending machine has 300 items IRL" on them.

The hard price was used for items that logically speaking has no major price jumps with market changes (Food and toys in this case) and to help with predictable price for the vend, also some items like a plastic fork and fortune cockle that has no place costing 200... or even 20..

File: Content.Server/VendingMachines/VendingMachineSystem.cs [hunk 1]
```diff
@@ -345,6 +345,12 @@ public void AuthorizedVend(EntityUid uid, EntityUid sender, InventoryType type,
 
             var totalPrice = ((int) price);
 
+            // This block exists to allow the MatketPrice flag to set a price
+            var priceMarket = _pricing.GetEstimatedMarketPrice(proto);
+            if (priceMarket == null || priceMarket == 0) { }
+            else { totalPrice = ((int) priceMarket); }
+            // This block exists to allow the MatketPrice flag to set a price
```
Comment (sTiKyt) [inline Content.Server/VendingMachines/VendingMachineSystem.cs:352]: Same "Matket" typo in comments here

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml [hunk 2]
```diff
@@ -2155,7 +2157,9 @@
     - Trash
   - type: SpaceGarbage
   - type: StaticPrice
-    price: 7.5
+    price: 5
+#  - type: MarketPrice
+#    price: 50
```
Comment (sTiKyt) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml:2162]: ?

**Comment by dvir001** [comment]
Removed some changes.

**Pull request closed without merging**